src/components/common/InputCurrencyTemplate.vue

```diff
@@ -22,7 +22,7 @@ const emit = defineEmits(['update:modelValue', 'click:max'])
 
 <template>
   <div
-    class="input-template px-4"
+    class="input-template px-4 grid-cols-[100%]"
```

Comment:
Duplication of `grid-template-columns: 100%` in styles.

Reply:
Fixed